database/migrations/2016_09_04_182835_create_notifications_table.php

```diff
@@ -14,7 +14,7 @@ public function up(): void
             $table->string('id')->primary();
             $table->string('type');
             $table->morphs('notifiable');
-            $table->text('data');
+            $table->json('data');
```

Comment:
Postgres should basically never use `json`. Always `jsonb`. There's a bunch of JSON operations in the code that depend on the database being able to understand the JSON, which is impossible with `json`.

Reply:
As far as I can see, they are identical in postgresql except for one being in plaintext which is slower. That's acceptable for BC.